src/app/app.module.js

```diff
@@ -0,0 +1,13 @@
+(function () {
```

Comment:
It is only useful when using variable in JavaScript. You do not need it with Angular.